CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - `srcSet` typo in the Image documentation.
 
+## [3.155.9] - 2022-02-03
+
+### Fixed
+- Enable 'isFullModeStyle, textMode, textPosition, textAlignment, blockClass' props available in the site Editor for Infocard component.
```

Comment:
This shouldn't be in this section. Please follow the instructions in our contributing guide: https://github.com/vtex-apps/store-discussion/blob/master/CONTRIBUTING.md.

Reply:
@victorhmp Updated